examples/design/EU-DEMO/EUDEMO_tests/test_tfcoils.py

```diff
@@ -0,0 +1,251 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
```

Comment:
Very minor, but the source file is `tf_coils`, so to match up, this file should probably be called `test_tf_coils`